test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -505,22 +505,39 @@ export default class PerfCompare extends Component {
                 curTestCI = parsedVariantsTest.metrics[j].value.CI;
 
                 // Get the metric's units
+                //Parse benchmark Variant and verify if the infos there
+                //get BenchmarkRouter & Metric files from server
+                let benchmarkMetricRequest = await fetch( `/api/getBenchmarkParsingInfo`, {
+                        method: 'get'
+                    }
+                );
+                let benchmarkMetricVariants = await benchmarkMetricRequest.json();
+                var BenchmarkMetricRouter = benchmarkMetricVariants['benchmarkMetricRouter'];
+                var BenchmarkMetric = benchmarkMetricVariants['benchmarkMetric'];
+                try {
+                    benchmarkProps = BenchmarkMetric[BenchmarkMetricRouter[curVariantData["benchmark"]][curVariantData["variant"]]]["metrics"];
+                } catch (routerNotFoundError) {
```

Comment:
We should create a library under `/src/utils` called `perf.js` and move this code there since it's used in multiple perf tools: Perf Compare and Tabular View. That library function should do everything and return `benchmarkProps` since that's the only thing that's needed here.